cmake/ROCMExportTargetsHeaderOnly.cmake

```diff
@@ -95,11 +95,12 @@ function(rocm_export_targets_header_only)
 
     if(PARSE_TARGETS)
         rocm_write_package_template_function(${CONFIG_TEMPLATE} include "\${${PACKAGE_NAME}_TARGET_FILE}")
-# Disabled for PyTorch
-#        foreach(NAME ${PACKAGE_NAME} ${PACKAGE_NAME_UPPER} ${PACKAGE_NAME_LOWER})
-#            rocm_write_package_template_function(${CONFIG_TEMPLATE} set ${NAME}_LIBRARIES ${PARSE_TARGETS})
-#            rocm_write_package_template_function(${CONFIG_TEMPLATE} set ${NAME}_LIBRARY ${PARSE_TARGETS})
-#        endforeach()
+        if(ROCM_EXPORT_PARSE_TARGETS_LIBRARY)
```

Comment:
This will not work, as we have to selectively build the package for PyTorch. The switch needs to be cmake exports regardless of the build option.

Reply:
Moved forward with the discussed idea.